### PR TITLE
merge replace-pty-js branch and fix tmux 2.6 scrolling issue

### PR DESCRIPTION
**Pull request by nightwing**
- mereges replace-pty-js branch which uses node-pty-prebuilt and tmux 2.2
- updates tmux version check to install tmux 2.2 if the existing tmux version is newer

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -308,14 +320,16 @@ compile_tmux(){
...
-  echo ":N.B: This will take a while. To speed this up install tmux 1.9 manually on your machine and restart this process."
+  echo ":N.B: This will take a while. To speed this up install tmux 2.2 manually on your machine and restart this process."
   
   echo ":Downloading Libevent..."
-  DOWNLOAD https://raw.githubusercontent.com/c9/install/master/packages/tmux/libevent-2.0.21-stable.tar.gz libevent-2.0.21-stable.tar.gz
+  # DOWNLOAD https://raw.githubusercontent.com/c9/install/master/packages/tmux/libevent-2.0.21-stable.tar.gz
+  DOWNLOAD https://github.com/libevent/libevent/releases/download/release-2.0.22-stable/libevent-2.0.22-stable.tar.gz libevent-2.0.22-stable.tar.gz
   echo ":Downloading Ncurses..."
-  DOWNLOAD https://raw.githubusercontent.com/c9/install/master/packages/tmux/ncurses-5.9.tar.gz ncurses-5.9.tar.gz
+  DOWNLOAD https://github.com/c9/install/raw/replace-pty-js/packages/tmux/ncurses-6.0.tar.gz ncurses-6.0.tar.gz
```
Comment (keyboardDrummer) [inline install.sh]: Consider copying the package binary to the master branch as well, and downloading from there

File: install.sh [hunk 2]
```diff
@@ -271,21 +283,21 @@ node(){
...
   echo ":Compiling libevent..."
-  tar xzf libevent-2.0.21-stable.tar.gz
-  rm libevent-2.0.21-stable.tar.gz
-  cd libevent-2.0.21-stable
+  tar xzf libevent-2.0.22-stable.tar.gz
+  rm libevent-2.0.22-stable.tar.gz
+  cd libevent-2.0.22-stable
   echo ":Configuring Libevent"
-  ./configure --prefix="$C9_DIR/local"
+  ./configure --disable-shared --prefix="$C9_DIR/local"
```
Comment (keyboardDrummer) [inline install.sh]: I don't know what this change is for, but I trust you know what you're doing

File: install.sh [hunk 3]
```diff
@@ -295,11 +307,11 @@ compile_tmux(){
...
   echo ":Compiling tmux..."
-  tar xzf tmux-1.9.tar.gz
-  rm tmux-1.9.tar.gz
-  cd tmux-1.9
+  tar xzf tmux-2.2.tar.gz
+  rm tmux-2.2.tar.gz
+  cd tmux-2.2
   echo ":Configuring Tmux"
-  ./configure CFLAGS="-I$C9_DIR/local/include -I$C9_DIR/local/include/ncurses" CPPFLAGS="-I$C9_DIR/local/include -I$C9_DIR/local/include/ncurses" LDFLAGS="-static-libgcc -L$C9_DIR/local/lib" LIBEVENT_CFLAGS="-I$C9_DIR/local/include" LIBEVENT_LIBS="-static -L$C9_DIR/local/lib -levent" LIBS="-L$C9_DIR/local/lib/ncurses -lncurses" --prefix="$C9_DIR/local"
+  ./configure CFLAGS="-I$C9_DIR/local/include -I$C9_DIR/local/include/ncurses" LDFLAGS="-static-libgcc -L$C9_DIR/local/lib" --prefix="$C9_DIR/local"
```
Comment (keyboardDrummer) [inline install.sh]: I don't know what this change is for, but I trust you know what you're doing

File: install.sh (hunk 2 above)
Comment (nightwing) [reply install.sh]: tmux changes were done by @mutahhir, and have been tested on v4

File: install.sh (hunk 1 above)
Comment (nightwing) [reply install.sh]: Will update after merging this cr.

**Pull request merged**